### DISCUSSION TITLE
Question about local thumbnails?

### DISCUSSION THREAD

**Discussion by WernerBuchert** [FriendsOfShopware/FroshPlatformThumbnailProcessor#117] [Q&A] [answered]
### Question

Good morning,

we only have two question's and that is that we have installed both the FroshPlatformBunnycdnMediaStorage and the FroshPlatformThumbnailProcessor and set the first plugin to activate the switch "Replicate bunnyCDN operations locally", so that the product image is also stored locally. 
With this setting and the 2nd plugin for the thumbnails, it now saves the original product image locally, but does not generate any thumbnails locally, is our understanding correct?

We know that with this setting, i.e. that we also save the image locally, we lose a bit of the advantage of saving local storage space, but we are not sure how reliable BunnyCDN is. What are your experiences and recommendations regarding storage zones, how many should we use to be reasonably safe?

Best regards, 
Werner Buchert.

**Comment by tinect** [comment, marked as answer]
Hey @WernerBuchert 

It is the main feature of FroshPlatformThumbnailProcessor that it does not create any thumbnail on disk.

**BunnyCDN**
It depends on your shop how much storage zones you need.
But it is recommended to have a minimum of two. The more, the more safety.

Your customers are just from Central Europe?
- use Falkenstein and US East (mainly for Google 😆 but also for failover)

**Comment by WernerBuchert** [comment]
Hey,

okay, thank's for the answer, it's now clear.
And thank's for this 2 nice plugin's !

Best regards,
Werner Buchert
